### PR TITLE
Fix regressionerrors

### PR DESCRIPTION
**Pull request by schoffelen**
This PR addresses 2 regression errors caused by the improved source2volume conversion.

### CODE REVIEW

File: test/test_ft_qualitycheck.m [hunk 1]
```diff
@@ -4,6 +4,8 @@
 % WALLTIME 00:30:00
 % DEPENDENCY ft_qualitycheck
 
+ft_hastoolbox('obsolete', 1);
```
Comment (robertoostenveld) [inline test/test_ft_qualitycheck.m:7]: should this not be `compat/obsolete` just like in test_issue623?

File: utilities/ft_checkdata.m [hunk 2]
```diff
@@ -1456,6 +1456,10 @@
     % reshape the first dimension
     tmp = getsubfield(data, fn{k});
     data.(fn{k}) = reshape(tmp, [data.dim dimsiz(2:end)]);
+  elseif startsWith(dimord, '{pos}')
```
Comment (robertoostenveld) [inline utilities/ft_checkdata.m:1459]: should we then also plan for `{pos_pos}`, which would be a 6D cell array?

Comment (robertoostenveld) [reply utilities/ft_checkdata.m:1459]: btw, reshaping the cell-array should indeed be fine

File: test/test_ft_qualitycheck.m (hunk 1 above)
Comment (schoffelen) [reply test/test_ft_qualitycheck.m:7]: yes, that's right. I was too rash

File: utilities/ft_checkdata.m (hunk 2 above)
Comment (schoffelen) [reply utilities/ft_checkdata.m:1459]: We could build in support for '{pos_pos}', let's see what can of worms will be opened

Comment (schoffelen) [reply utilities/ft_checkdata.m:1459]: OK, I think it's done.

**Comment by schoffelen** [comment]
I am curious whether the daily test batch runs through, I hope that now only test_data2bids will continue to fail...

**Pull request merged**